cloud-common/src/main/java/com/sequenceiq/cloudbreak/domain/CommonStatus.java

```diff
@@ -0,0 +1,6 @@
+package com.sequenceiq.cloudbreak.domain;
```

Comment:
I think that the common project packages shall also reflect that they are in common therefore changing the package  from com.sequenceiq.cloudbreak.domain to  com.sequenceiq.cloudbreak.common.type would be better